LICENSE.txt

```diff
@@ -1,4 +1,4 @@
-PowerShell 6.0
```

Comment:
@SteveL-MSFT, your last commit had 1 failures in `PowerShell-CI-static-analysis`
Verify Markdown Links.Verify links in /home/vsts/work/1/s/CHANGELOG/README.md.preview.md should work
```powershell
Tool reported URL as unreachable.
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 124
124:                             throw "Tool reported URL as unreachable."

```